src/plugins/data_source_management/public/mocks.ts

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { throwError } from 'rxjs';
+import { SavedObjectsClientContract } from 'opensearch-dashboards/public';
+import { AuthType } from './types';
+import { coreMock } from '../../../core/public/mocks';
+
+/* Mock Types */
+
+export const docLinks = {
+  links: {
+    noDocumentation: {
+      indexPatterns: {
+        introduction: '',
+      },
+      scriptedFields: {},
+    },
+  },
+};
+
+const createDataSourceManagementContext = () => {
+  const {
+    chrome,
+    application,
+    savedObjects,
+    uiSettings,
+    notifications,
+    overlays,
+  } = coreMock.createStart();
+  const { http } = coreMock.createSetup();
+
+  return {
+    chrome,
+    application,
+    savedObjects,
+    uiSettings,
+    notifications,
+    overlays,
+    http,
+    docLinks,
+    setBreadcrumbs: () => {},
+  };
+};
+
+export const mockManagementPlugin = {
+  createDataSourceManagementContext,
+  docLinks,
+};
+
+/* Mock data responses - JSON*/
+export const getDataSourcesResponse = {
+  savedObjects: [
+    {
+      id: 'test',
+      type: 'data-source',
+      description: 'test datasource',
+      title: 'test',
+      get(field: string) {
+        const me: any = this || {};
+        return me[field];
+      },
+    },
+    {
+      id: 'test2',
+      type: 'data-source',
+      description: 'test datasource2',
+      title: 'test',
+      get(field: string) {
+        const me: any = this || {};
+        return me[field];
+      },
+    },
+    {
+      id: 'alpha-test',
```

Comment:
just curious to know if the name means anything, alpha-test vs beta-test.  Also seems beta-test is been not used yet?

Reply:
This is just mock data and just used for testing purposes .. Names mean nothing

Reply:
ok, feels a little confusing to me given it has state related names, if doesn't apply to our case maybe just keep consistency with previous naming.